tests/core/pyspec/eth2spec/test/altair/transition/test_slashing.py

```diff
@@ -0,0 +1,133 @@
+import random
+from eth2spec.test.context import (
+    always_bls,
+    fork_transition_test,
+)
+from eth2spec.test.helpers.constants import PHASE0, ALTAIR
+from eth2spec.test.helpers.attester_slashings import (
+    get_valid_attester_slashing,
+)
+from eth2spec.test.helpers.proposer_slashings import (
+    get_valid_proposer_slashing,
+)
+from eth2spec.test.helpers.fork_transition import (
+    do_altair_fork,
+    state_transition_across_slots_with_ignoring_proposers,
+    transition_until_fork,
+    transition_to_next_epoch_and_append_blocks,
+)
+from eth2spec.test.helpers.inactivity_scores import (
+    slash_some_validators_for_inactivity_scores_test,
+)
+
+
+@fork_transition_test(PHASE0, ALTAIR, fork_epoch=1)
+def test_transition_with_one_fourth_slashed_active_validators_pre_fork(
+        state, fork_epoch, spec, post_spec, pre_tag, post_tag):
+    """
+    1/4 validators are slashed but still active at the fork transition.
+    """
+    # slash 1/4 validators
+    slashed_indices = slash_some_validators_for_inactivity_scores_test(
```

Comment:
see note about the refactoring of this function -- it may make more sense to instead use/modify `helpers/random.py::slash_some_validators` for this purpose

Reply:
if you want to keep this function like you have it, then we should change the name to something more generic than "for_inactivity_scores_test"